### ISSUE TITLE
Great Tool for Compettitions, wanna help to translate it to spanish.

### ISSUE THREAD

**Issue by VAbgta** [geoffhumphrey/brewcompetitiononlineentry#1060]
HI, I'm a Professional brewer and a BJCP judge and using recently BCOE&M I love it, but I live in Colombia and it will be great to have a Spanish version for Latin American competitions. So if you need help translating stuff I will be happy to help.

**Comment by geoffhumphrey** [comment]
Absolutely! I can't believe I missed this from months ago. My apologies for that. Please contact me if you'd like to serve as Spanish language translator for the software. As it stands today, we've got people working on Brazilian Portuguese and Czech languages.

**Comment by VAbgta** [comment]
Hi Geoff, yes i can help in that, i already do a big part of the translation, at least the part that i can change, my code knowledge is basically 0 hehe, but well i make it, and is usable right know, the only thing is that i cannot find where to change some parts and there is where i need help. If you want you can send me an email to aramarth@gmail.com, great software for competitions I’m already using it in a competition in two weeks.

**Comment by geoffhumphrey** [comment]
Sure thing. I'll get in touch with you next week. I did remove your email from your comment since I saved it already. No need for SPAM. If you wouldn't mind, please post your language file in a zip file here.

**Comment by VAbgta** [comment]
Ok great, here is the file that im using now that have almost everything in
Spanish i need to work it a little more on it.

In the style entry text that is on line 1058 i cannot make it work on the
final entry registration form, it always is show on english.

When you have time let me know how i can help you.

Cheers,

Daniel López

**Comment by geoffhumphrey** [comment]
@VAbgta - it looks like those style-related translations are not being utilized in the code base, hence why you're not seeing them. As of now, I can't remember why, but I'll investigate.

**Comment by geoffhumphrey** [comment]
@VAbgta - Also, if you would compress (zip) your files and upload them here on GitHub, I would appreciate it!

**Comment by VAbgta** [comment]
Hi Geoff, sorry for answer two days later i was a little complicated with
work on the brewery, let me finish the Spanish translation completely and i
will upload or send it to you.

If you need something else let me know.
